packages/Tokens/src/tokens.yaml

```diff
@@ -147,6 +147,7 @@ card:
 popover:
   border-radius: $card--border-radius
   shadow: 0 2px 4px rgba(0, 0, 0, 0.1)
+  maxWidth: 320
```

Comment:
`max-width`

Reply:
Actually, should be `max-width: 320px`.  That means adjusting the code too though.

Reply:
Actually the popover component actually would use this token without the "px" hence why i left it off... wdyt?

Reply:
Now passing "320px"